src/utils/components/AddBootableVolumeModal/components/VolumeMetadata/components/InstanceTypeDrilldownSelect/components/UserInstanceTypeMenu/UserInstanceTypeMenu.tsx

```diff
@@ -1,29 +1,42 @@
 import React, { Dispatch, FC, SetStateAction, useMemo, useState } from 'react';
 
+import { InstanceTypes } from '@catalog/CreateFromInstanceTypes/state/utils/types';
+import { VirtualMachineInstancetypeModelGroupVersionKind } from '@kubevirt-ui/kubevirt-api/console';
+import { isRedHatInstanceType } from '@kubevirt-utils/components/AddBootableVolumeModal/components/VolumeMetadata/components/InstanceTypeDrilldownSelect/utils/utils';
 import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
 import { VirtualMachineClusterInstancetypeModelGroupVersionKind } from '@kubevirt-utils/models';
+import { getName } from '@kubevirt-utils/resources/shared';
 import { isEmpty } from '@kubevirt-utils/utils/utils';
 import { ResourceIcon } from '@openshift-console/dynamic-plugin-sdk';
 import { MenuInput, MenuItem, SearchInput } from '@patternfly/react-core';
 
 type UserInstanceTypeMenuProps = {
+  allInstanceTypes: InstanceTypes;
   items: string[];
   selected: string;
   setSelected: Dispatch<SetStateAction<string>>;
 };
 
-const UserInstanceTypeMenu: FC<UserInstanceTypeMenuProps> = ({ items, selected, setSelected }) => {
+const UserInstanceTypeMenu: FC<UserInstanceTypeMenuProps> = ({
+  allInstanceTypes,
+  items,
```

Comment:
i dont think we need items and allinstancetypes, only one of them.

Reply:
Fixed!